### ISSUE TITLE
Model and loss function details

### ISSUE THREAD

**Issue by rozeappletree** [koushikvikram/multimodal-image-retrieval#14]
Hi, can you please let me know the model that you are using for training and also the multimodal loss function that you are using.

**Comment by koushikvikram** [comment]
Hi @INF800 , thank you for your interest in our project!

We use ResNet50 with Binary Cross Entropy Loss for regressing the Word2Vec Embeddings from images.

Please let me know if you need any more information.
